test/integration/default/controls/desktop_and_screensaver.rb

```diff
@@ -0,0 +1,35 @@
+title 'desktop & screen saver'
```

Comment:
I would prefer `and` over `&`.

Reply:
Why I wrote it with the ampersand:

![screenshot 2018-03-24 15 43 19](https://user-images.githubusercontent.com/8458267/37869719-1de81f2e-2f7a-11e8-982e-90aace7c74e9.png)